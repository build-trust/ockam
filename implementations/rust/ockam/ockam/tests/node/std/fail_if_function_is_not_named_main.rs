#[ockam::node]
fn foo(c: ockam::Context) {
    c.stop().await.unwrap();
}
