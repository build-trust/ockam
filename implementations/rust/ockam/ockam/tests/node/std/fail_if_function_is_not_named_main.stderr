error: the function name must be `main`
 --> tests/node/std/fail_if_function_is_not_named_main.rs:2:4
  |
2 | fn foo(c: ockam::Context) {
  |    ^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/node/std/fail_if_function_is_not_named_main.rs:4:2
  |
4 | }
  |  ^ consider adding a `main` function to `$DIR/tests/node/std/fail_if_function_is_not_named_main.rs`
